### DISCUSSION TITLE
Iterating over a range broken

### DISCUSSION THREAD

**Discussion by Kasparsu** [alpinejs/alpine#3330] [7. Bugs (archived)]
Iterating over a range seems to be broken

[](https://codepen.io/kasparsu/pen/gOKygvy)
example 

https://alpinejs.dev/directives/for#iterating-over-a-range
![image](https://user-images.githubusercontent.com/3220262/206783083-1a077cc2-9203-433f-a8ea-db40f10fd920.png)

**Comment by ekwoka** [comment]
What behavior are you expecting and what behavior are you getting? Are there any errors?

can you also show the whole component not just the expression?
